### ISSUE TITLE
Make Template helpers optional

### ISSUE THREAD

**Issue by jankapunkt** [VeliovGroup/Meteor-Internationalization#8]
We use a custom Template helper and would like to omit the internal one, for example by passing `helperName: null` and `helperSettingsName: null`. If `null` is passed then there would be no helper created.

Example would be:

```js
      if (typeof Template !== 'undefined' && Template !== null) {
        if (this.helperName !== null) {
          /**
           * @summary Main `i18n` template helper
           */
          Template.registerHelper(this.helperName, function () {
            return self.get.apply(self, arguments);
          });
        }

        if (this.helperSettingsName !== null) {
          /**
           * @summary Settings `i18n` template helper, might be used to build language switcher (see demo folder).
           */
          Template.registerHelper(this.helperSettingsName, function () {
            return self.getSetting.apply(self, arguments);
          });
        }
      }
```

what do you think?

**Comment by dr-dimitru** [comment]
@jankapunkt sounds good, I'm glad you're using it.
Would you send a PR?

**Comment by jankapunkt** [comment]
@dr-dimitru great :-) See #9

**Comment by dr-dimitru** [comment]
@jankapunkt implemented in [📦 v3.2.1](https://github.com/veliovgroup/Meteor-Internationalization/releases/tag/3.2.1)
